test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTest.cs

```diff
@@ -630,4 +642,4 @@ private class TestOptionsManager : IOptions<AntiforgeryOptions>
             public AntiforgeryOptions Value { get; set; } = new AntiforgeryOptions();
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I assume this is :see_no_evil:, don't know why GitHub trimmed off the "old" line